Elsewhen/ContentView.swift

```diff
@@ -64,6 +64,9 @@ struct ContentView: View {
             }
             showShareSheet = .init(items: activityItems)
         }
+        .onAppear {
+            selectedTab = UserDefaults.shared.defaultTab
```

Comment:
Does this need to be in the shared app group? Seems like it's only useful in-app?

Reply:
You're right, but I couldn't remember what the alternative was 😅 `standard` ?